documentation/manual/source/_theme.indigo/layout.html

```diff
@@ -152,83 +147,56 @@
 {%- block footer %}
 <div id="footer-wrapper">
   <footer id="footer">
-    <div id="supported-by">
-      <a href="http://1and1.com">
-        <img src="http://resources.qooxdoo.org/images/1and1.png">
-      </a>
-      <p class="claim">
-        Brought to you by <a href="http://1and1.com">1&amp;1</a>, one of the world's leading web hosts
-      </p>
-      <p class="legal">
-        © 1&amp;1 Internet AG<br>
-        <a href="http://qooxdoo.org/legal">Legal notice</a>
-      </a>
-    </div>
     <div id="sitemap">
       <div>
         <h3><a href="http://qooxdoo.org/">Home</a></h3>
         <ul>
           <li>
-            <a href="http://qooxdoo.org/demos">Demos</a>
-          </li>
-          <li>
-            <a href="http://qooxdoo.org/download">Download</a>
-          </li>
-          <li>
-            <a href="http://news.qooxdoo.org">Blog</a>
+            <a class="" href="http://qooxdoo.org/demos.html">demos</a>
           </li>
           <li>
-            <a href="http://bugs.qooxdoo.org">Bugs</a>
```

Comment:
It looks like we lost Bugs, which should probably point to the github Issues page